hack/show-edges.py

```diff
@@ -262,13 +262,16 @@ def load_blocks(versions, revision=None, directory='blocked-edges'):
 
 
 def get_blocked(edges, blocks, architecture):
-    blocked = set()
+    blocked = {}
     for from_version, to_version in edges:
         for block in blocks:
             if to_version == block['to']:
                 regexp = re.compile(block['from'])
                 if regexp.match('{}+{}'.format(from_version, architecture)):
-                    blocked.add((from_version, to_version))
+                    key = (from_version, to_version)
+                    if key not in blocked:
+                        blocked[key] = set()
+                    blocked[key].add(block.get('name'))
```

Comment:
nit: Alternatively, we can use [defaultdict](https://docs.python.org/3/library/collections.html#collections.defaultdict) here, it'll automatically initialize an empty `set()` if there exists no key.